spec/lib/oauth/token_spec.rb

```diff
@@ -18,16 +18,22 @@ module OAuth
         end
 
         it 'accepts anything that responds to #call' do
+          skip 'No assertion.'
+
           expect(method).to receive(:call).with(request)
           Token.from_request request, method
         end
 
         it 'delegates methods received as symbols to Token class' do
+          skip 'No assertion.'
```

Comment:
Same. It seems the `skip`s were actually not needed, but I would like a rewrite of these specs to denote better the assertion after the execution. It is a common confusion.